yt/docs/README.md

```diff
@@ -1,208 +1,434 @@
 
-* вида доступа (чтение, запись и т.д.);
-* пользователя, инициировавшего запрос;
-* объекта, к которому доступ запрашивается.
+When adding a link into an article that is located in the `_includes` folder, keep the following in mind:
 
-## Инфоблоки
+- A link should be relative to the `_includes` folder, but not to the original oneю Цhat does it mean, see the example below.
+- A link should refer to the article located in the original folder, but not in the `_includes` folder.
 
-Информационные примечания задаются: `{% note info %} ... {% endnote %}`. Само примечание начинается с большой буквы.
+For an example, take a look at the [Query Tracker](https://github.com/ytsaurus/ytsaurus/edit/main/yt/docs/en/user-guide/query-tracker.md) article. Its source is stored in the `en/user-guide/query-tracker.md` file.
 
 ```
-{% note info "Примечание" %}
+|-- ytsaurus/yt/docs
+    |-- en/
+        |-- user-guide/             # The original folder.
+            |-- query-tracker.md    
+        |-- dynamic-tables/  
+            |-- dyn-query-language.md
+        |-- _includes/              # The "_includes" folder, where the common content is held.
+           |-- user-guide/
+               |-- query-tracker.md
+           |-- dynamic-tables/  
+               |-- dyn-query-language.md
+                   
+```
 
-Права на создание и удаление групп в данный момент не выдаются пользователям и осуществляются через форму или вкладку ACL узла в веб-интерфейсе. Добавление/удаление пользователей в группы делегированы ответственным через IDM.
+Text of this article is included from the `en/_includes/user-guide/query-tracker.md` file:
 
-{% endnote %}
+```
+{% include [Query Tracker](../_includes/user-guide/query-tracker.md) %}
 ```
 
-Предупреждающие примечания задаются: `{% note warning %} ... {% endnote %}`. Само примечание начинается с большой буквы.
+Now, open the `en/_includes/user-guide/query-tracker.md` file. There you can find the link to the "YT Query Language" article:
 
 ```
-{% note info "Примечание" %}
+Currently supported execution engines include: [YT QL](../../user-guide/dynamic-tables/dyn-query-language.md).
+```
+
+The `dyn-query-language.md` article also reuses content, same as `query-tracker.md`. So, as you can see in the code above:
+- The "YT QL" link refers to the article located in the `en/user-guide/dynamic-tables/` folder, but not the `en/_includes/user-guide/dynamic-tables/`.
+- This link is set relativelly to the `_includes` folder.
 
-Пользователи и группы располагаются в одном пространстве имен, то есть имена пользователей и групп не могут совпадать.
+### Headers
 
-{% endnote %}
+Headers should always have anchors:
+
+```
+## Users, Groups { #users-groups }
+```
+
+
+### Lists
+
+Every item in an ordered list starts with a capital letter and ends with a period, while items in an unordered list start with a lowercase letter and end with a semicolon; the last item in the list always ends with a period.
+
```

Comment:
Worth adding when we use ordered lists, and when we use unordered lists.
(Probably when order matters we use ordered, and unordered otherwise. Rule with semicolon is strange in this case)